migrations/20241110010322_adds_game_version_minmax.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE loader_fields SET min_val = 1 WHERE field = 'game_versions';
+UPDATE loader_fields SET max_val = 20 WHERE field = 'game_versions';
```

Comment:
why 20? before we didn't have a limit for game versions, why add one now?

Reply:
Ah fair enough.